### DISCUSSION TITLE
Cloud Deployment

### DISCUSSION THREAD

**Discussion by VolodymyrBlueZebra** [apache/camel-karavan#1395] [Q&A]
### Ask a question

Hi Everyone,
In the repository description, there is a ‘deployer’ section, which states that it contains scripts for various platforms. Could someone share a short guide on how to deploy the platform on AWS?
Thank in advance

### Variant

None

### Container Management (if applicable)

None

### Operating System (if applicable)

None

### Version

None

**Comment by mgubaidullin** [comment]
Deploy Karavan on docker or kubernetes. Karavan deploys integrations in the same environment
